docs/os_boot_serial_console.md

```diff
@@ -0,0 +1,130 @@
+Enabling OS boot serial console and setting connection parameters
+=================================================================
+
+# Intro
+-------
+* Instructions is written for Voyage Linux, but steps showed there should work
+for another Linux distributions
+* Used serial connection parameters: `baudrate`: 115200, `data bits`: 8, 
+`parity`: none, `stop bits`: 1
+* Used text editor is `nano`, but it can be any another text editor e.g. `vi`
+* Root user is recommended
+
+# Enabling sending OS boot logs via serial console
+--------------------------------------------------
+
+Often sending OS boot logs via serial console is disabled as default setting.
+To enable that function `menu.lst` file has to be edited. To do that follow
+the steps below.
+
+1. Login to Linux Voyage. Default login/password: `root`/`voyage`.
+
+2. After logging to Voyage Linux filesystem is mounted as `read-only`. To 
+change that type:
+
+```
+remountrw
+```
+
+3. Open `menu.lst` file with any text editor. E.g:
+
+```
+nano /boot/grub/menu.lst
+```
+4. Find the settings of configuration from grub menu which you choose. The 
+structure should look like:
+
+```
+title           Debian GNU/Linux, kernel 3.10.11
+root            (hd0,0)
+kernel          /boot/vmlinuz-3.10.11 root=UUID=f45cf8c7-311e-47d6-88d0-a3a8861f37be ro 
+initrd          /boot/initrd.img-3.10.11
+```
+
+5. Add to the kernel line following options:
+
+```
+vga=normal console=tty0 console=ttyS0,115200n8
```

Comment:
`vga=normal` and `console=tty0` shouldn't be needed to enable just the serial console